### ISSUE TITLE
managing a value?

### ISSUE THREAD

**Issue by wdcurry** [bcylin/QuickTableViewController#16]
Is there any provision within this very helpful repo to manage a value? A small textbox of numeric input for example?

**Comment by bcylin** [comment]
Thanks for your feedback. Is it like a `TextFieldRow` representing a cell with a text field?

I suppose it's possible to follow the same pattern the `SwitchRow` uses, but I don't have a good idea at the moment. Do you have any thoughts?

**Comment by wdcurry** [comment]
Not really. I was using your repo as a quick solution to a fuzzy settings view (meaning i wasn't sure what i wanted to "set" and didn't want to hard-code. Enjoy it's usage. Took me a while to get the code structure right for setting options to my system-wide variables (as i usually do this in a "setupUI" routine outside of the structural setup. I found no easy way to set options. I would suggest a hook into the parent of the options to allow an "injection" of a newly selected value, which can be immensely helpful at least to my style.

The value proposition here in this thread is nothing spectacular, just a textbox, possible with internal value range to negate the need to code a check of our own. CGFloat to allow us to dumb it down to Int if need be? Just thoughts, but i am sure it would help others! cheers!  nice work!!

**Comment by bcylin** [comment]
Not sure if this is what you were looking for, but this is how I usually set up options with the system-wide variables. It's not very obvious at the first glance so I'm still looking for a better way to set it up. 

Can you describe a little bit more about how the textbox works in code? It sounds like an alternative solution to this problem.

```swift
private lazy var radioSection: RadioSection = {
  let savedValue: Value = // the setting saved somewhere, e.g. in UserDefaults
  let options = Value.availableValues.map { value in
    OptionRow(
      title: String(describing: value),
      isSelected: value == savedValue,
      action: { [weak self] in
        if option = $0 as? OptionRowCompatible, option.isSelected {
          self?.updateValue()
        }
      }
    )
  }
  return RadioSection(title: "...", options: options)
}()

// ...

private func updateValue() {
  if let selectedIndex = radioSection.indexOfSelectedOption {
    let selectedValue = Value.availableValues[selectedIndex]
    // Save the selected value
  }
}
```
